components/automate-gateway/api/iam/v2/tokens.proto

```diff
@@ -38,7 +68,18 @@ service Tokens {
     option (chef.automate.api.policy).action = "get";
     option (chef.automate.api.iam.policy).resource = "iam:tokens:{id}";
     option (chef.automate.api.iam.policy).action = "iam:tokens:get";
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      tags: "tokens"
+    };
   };
+
+  /*
+  Update a token
+
+  This PUT operation will overwrite all fields excepting ID, timestamps, and value,
+  including those omitted from the request. Include `name`, `active` and `projects`
+  in the request.
```

Comment:
I find it difficult to understand what these two sentences are conveying, and I am familiar with the code. I think users would be lost.